### ISSUE TITLE
Nice to have: blurrying technique on image with text on top

### ISSUE THREAD

**Issue by alberto-campos** [dbykovsky/sumofusM#99]
On the image listing, it'd be nice to apply the following blurrying technique to make our text more legible:
http://trickyandroid.com/advanced-blurring-techniques/

**Comment by dbykovsky** [comment]
Good feature. I'll take a look at that. We should be consistent and have
this on list view and detailed view.

Alberto, how is Fragment stuff? Do you think you can do that?

**Comment by alberto-campos** [comment]
Yes. I'll continue working in the new campaign fragment during my lunch break.

**Comment by dbykovsky** [comment]
Hey Alberto,

I'm not sure how far you are with the framents. May be you want to take a
look at this repo with quizes.
https://github.com/googlesamples/android-topeka

We may adopt it for out needs. We just need to collect text instead of just
click of fab button

**Comment by alberto-campos** [comment]
I tried that yesterday but it would completely change our flow and I didn't want to risk due to the short time that we have. So, for today, I'm making this view similar to the Sign Petition view.

We should definetely move to that if we make it to the next milestone.

**Comment by dbykovsky** [comment]
Oh, ok. Totally works for me.
